nucleus/payara-modules/opentracing-adapter/src/main/java/fish/payara/opentracing/tracer/Tracer.java

```diff
@@ -328,6 +331,20 @@ public Scope startActive(boolean bln) {
                 asChildOf(parent);
             }
             origin = scopeManager.activate(span, bln);
+            
+            if (requestTracing != null && !requestTracing.isTraceInProgress()) {
+                if (span.getSpanReferences().isEmpty()) {
+                    span.setEventType(EventType.TRACE_START);
+                } else {
+                    span.setEventType(EventType.PROPAGATED_TRACE);
+                    
+                    // Assume the first parent reference found has the correct traceId - would it ever not be?
```

Comment:
Seems like a left-over from development

Reply:
Currently on master, will be added to separate task